### PR TITLE
fixes #18117, #18118 - adds Puppet 4 to katello-backup and katello-remove

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

**Comment by ehelms** [comment]
Looks good -- @stbenjam double check all locations covered for P4?

File: katello/katello-backup [hunk 1]
```diff
@@ -111,6 +111,8 @@ else
     '/etc/pki/pulp',
     '/etc/pulp',
     '/etc/puppet',
+    '/etc/puppetlabs/puppet/ssl',
+    '/etc/puppetlabs/puppet/foreman_cache_data',
```
Comment (stbenjam) [inline katello/katello-backup]: Probably grab all of /etc/puppetlabs just like we grab all of /etc/puppet.  There's config files in there for agent + server.

Comment (netman2k) [reply katello/katello-backup]: I agreed with @stbenjam, we need to grab all of the /etc/puppetlabs directory. this directory contains puppet codes as well.

**Comment by cfouant** [comment]
@stbenjam - changes made. Also updated katello-remove to reflect /etc/puppetlabs rather than /etc/puppetlabs/puppet. Thanks!

**Review by stbenjam — APPROVED** [review]

**Comment by stbenjam** [comment]
Looks good to me!

**Comment by ehelms** [comment]
Thanks @cfouant !

**Pull request merged**